### DISCUSSION TITLE
Does it work with hue bulbs when they were powered off completly with a conventional wall switch?

### DISCUSSION THREAD

**Discussion by cswrd** [basnijholt/adaptive-lighting#626] [Q&A] [answered]
Does adaptive-lighting work when the bulbs are powered off completly, i.e. turned off completly with a  conventional switch (not a smart wall switch module or dimmer switch etc.)?
So when turned on do they lighten up in the corrct brightness for the current date time? Do they also keep getting brighter or darker throughout the day?
Are there any side effects like they start at full brightness for a second and become darker if needed a couple of seconds later?

**Comment by protyposis** [comment, marked as answer]
Yes, in my Hue + Zigbee2MQTT setup this works. As a side effect, there is a delay of a few seconds until Home Assistant recognizes the light being back online and on, and thus the adaptation is also delayed by the same amount.

**Reply by cswrd** [reply]
Thank you. I was hoping for no delay or something not noticeable, but it's good to know upfront.
Since this behaviour is very important for me, I'll do some research/tests with a different automation software first. One of them was designed specifically for conventional wall switches. Maybe they found an optimization in that regard.
